src/compiler/Parser/ParsedHierarchy/Expression/NumberExpression.ts

```diff
@@ -1,6 +1,8 @@
 import { Container as RuntimeContainer } from "../../../../engine/Container";
 import { Expression } from "./Expression";
 import { BoolValue, FloatValue, IntValue } from "../../../../engine/Value";
+import { asOrNull } from "../../../../engine/TypeAssertion";
+import { ParsedObject } from "../Object";
 
 export class NumberExpression extends Expression {
```

Comment:
Comment for later, it's named `Number` in upstream, even though this name makes more sense to me. To be documented later on.

Reply:
[Number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number) is a predefined class in js hence the new name. I'll add a comment in code.

Reply:
Works for me!